css/global.css

```diff
@@ -1,6 +1,7 @@
 @import './variables.css';
 @import './navigation-pages.css';
 @import './fast-links.css';
+@import './action-buttons.css';
```

Comment:
I cant' see this css file, did you forget to commit it?

Reply:
Correct I somehow failed to add it